Tools/ardupilotwaf/boards.py

```diff
@@ -558,6 +558,12 @@ def __init__(self):
         self.px4io_name = 'px4io-v2'
         self.with_uavcan = True
 
+class the_cube_green(px4_v3):
+    name = 'the-cube-green'
+    def __init__(self):
+        super(the_cube_green, self).__init__()
+        self.param_defaults = '../../../Tools/Frame_params/the-cube-green.param'
```

Comment:
We already have a file for this in frame_params that I believe this should use. Please direct it to this: https://github.com/ArduPilot/ardupilot/blob/master/Tools/Frame_params/Solo_AC35.param.  The new file in frame_params can be removed from the PR.

Reply:
Humn... I didn't notice they are actually the same. I've got these params from @proficnc... I'm not sure if he plans to do something different on the green cube.

Reply:
That param file is the one I sent him :).  It's actually not the right file anyway. I thought he wanted the full 3.5 defaults when he asked me for it, so that's what I sent him, and that's what that file is...  Just naked 3.5 defaults.  The existing file in frame_params is actually what you need for this, and probably what Philip was looking for when he asked for the file.